I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 21, in script
    "You've created a new Ren'Py game."
OSError: Couldn't find file 'voice/start_984ab428.ogg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 21, in script
    "You've created a new Ren'Py game."
  File "/Applications/renpy-8.1.1-sdk/renpy/ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/Applications/renpy-8.1.1-sdk/renpy/exports.py", line 1473, in say
    who(what, *args, **kwargs)
  File "/Applications/renpy-8.1.1-sdk/renpy/character.py", line 1352, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "/Applications/renpy-8.1.1-sdk/renpy/character.py", line 1003, in do_display
    display_say(who,
  File "/Applications/renpy-8.1.1-sdk/renpy/character.py", line 693, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Applications/renpy-8.1.1-sdk/renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-8.1.1-sdk/renpy/display/core.py", line 3579, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Applications/renpy-8.1.1-sdk/renpy/display/core.py", line 4376, in interact_core
    renpy.audio.audio.periodic()
  File "/Applications/renpy-8.1.1-sdk/renpy/audio/audio.py", line 1222, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 444, in raise_
  File "/Applications/renpy-8.1.1-sdk/renpy/audio/audio.py", line 1200, in periodic_thread_main
    periodic_pass()
  File "/Applications/renpy-8.1.1-sdk/renpy/audio/audio.py", line 1142, in periodic_pass
    c.periodic()
  File "/Applications/renpy-8.1.1-sdk/renpy/audio/audio.py", line 534, in periodic
    topf = load(filename)
  File "/Applications/renpy-8.1.1-sdk/renpy/audio/audio.py", line 72, in load
    rv = renpy.loader.load(fn, directory="audio")
  File "/Applications/renpy-8.1.1-sdk/renpy/loader.py", line 833, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'voice/start_984ab428.ogg'.

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.1.1.23060707
avdemo 1.0
Sun Jul 30 02:56:01 2023
